scripts/goggle_classifier.py

```diff
@@ -22,7 +22,7 @@
 # 80/20 training/validation split
 VAL_SPLIT = .2
```

Comment:
this is the second time ive seen this constant defined. maybe we should include them in a more centralized place.